docs/admin-guide/troubleshooting.md

```diff
@@ -92,9 +92,128 @@ The steps below are applicable for Severity ("**Sev**") 1 & 2 level incidents. S
 
 After receiving a new alert or ticket, spend a few minutes (&lt; 5m) do a preliminary investigation to identify the reproducibility and impact of the incident. Once confirmed as an issue, the formal incident response process is:
 
-### Hypothesis Development
+### Observing the environment
 
-### Hypothesis Testing
+:1: Start with checking general cluster health
+
+:2: Check recent changes that may have had an impact
+
+:3: Check recent trends in consumption metrics:
+
+* Has a threshold been reached? A threshold can be both fixed (e.g. memory) and cumulative (disk space).
+
+:4: Check performance metrics
+
+* Are metrics within bounds?
+* Are any metrics abnormal?
+
+:5:
+
+### Hypothesis Development & Testing
+
+- What do I think is the problem?
+- What would it look like if I were right about what the problem is?
+- Could anything other problem present in this way?
+- Of all the possible causes I've identified, are any much more likely than others?
+- Of all the possible causes I've identified, are any much more costly to investigate and fix than others?
+- How can I prove that, of all the problems that present in this way, the problem that I think is the problem is in fact the problem?
+- How can I test my guesses in a way that, if I'm wrong, I still have a system that is meaningfully similar to the one that I was presented with at the start?
+
+Why even talk about 'hypotheses' rather than 'ideas' or guesses'? Taking on more formal scientific language is a signal of connection to and create a bridge for a larger set of ideas and techniques about evidence and progress in knowledge. Because of the cache scientific ideas hold, it's important to understand that much of theory is constructed in hindsight – science, like engineering is a messy, human business and there is no perfect method for developing and testing hypotheses. Whatever is included here will necessarily leave much out, and there will always be cases that are not covered or in which the advice below misleads – nevertheless, it has been our experience that the following set of questions are a useful [heuristic](https://en.wikipedia.org/wiki/Heuristic).
```

Comment:
creating?
is cache the right word here?

Reply:
Meant to be `caché` lol

Not sure what you mean by `creating?`

Reply:
'Taking on more formal scientific language is a signal of connection to and create a bridge' - creates a bridge, creating a bridge?  The tense as it stands is wrong.  You maybe want to break that sentence up a bit, it's a lot to parse in one thought